### ISSUE TITLE
Unable to import ncbitaxon taxslim-disjoint-over-in-taxon.owl

### ISSUE THREAD

**Issue by gaurav** [ExposuresProvider/cam-pipeline#99]
When running `ROBOT_JAVA_ARGS="-Xmx96G -XX:+UseParallelGC" robot mirror -i ontologies.ofn -d mirror -o mirror/catalog-v001.xml` in https://github.com/ExposuresProvider/cam-pipeline/pull/98, I get the following error:

```
org.semanticweb.owlapi.model.UnloadableImportException: Could not load imported ontology: <http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim-disjoint-over-in-taxon.owl> Cause: Ontology already exists. OntologyID(OntologyIRI(<http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl>) VersionIRI(<http://purl.obolibrary.org/obo/ncbitaxon/releases/2023-06-16/subsets/taxslim.owl>))
```

This appears to be because both of those ontologies are identified with the identifier `http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl`.

Comparing those files, it looks like they are identical exception for one additional RO term in http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim-disjoint-over-in-taxon.owl. Since this is the case, I will comment out http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl for now. But we might want to contact ncbitaxon and figure out which of these files we should use.

**Comment by balhoff** [comment]
@gaurav this is a temporary issue that I reported earlier today: https://github.com/obophenotype/ncbitaxon/issues/76

**Comment by gaurav** [comment]
@balhoff Nice! I've added a note to https://github.com/ExposuresProvider/cam-pipeline/pull/100 so we wait for that bug to be fixed upstream before updating and merging that PR.

**Comment by gaurav** [comment]
Thanks, @anitacaron, this works now! Closed in PR #100.
